tests/PKSim.Tests/Presentation/MultiParameterEditPresenterSpecs.cs

```diff
@@ -250,6 +248,59 @@ public void should_hide_the_distribution_if_unvisible()
       }
    }
 
+   public class When_editing_a_set_of_parameters : concern_for_MultiParameterEditPresenter
+   {
+      private IParameter _visibleParameter;
+      private IParameter _hiddenUnchangedParameter;
+      private IParameter _hiddenIsFixedParameter;
+      private IParameter _hiddenIsNotDefaultParameter;
+      private List<ParameterDTO> _allParameterDTO;
+      private List<IParameter> _displayedParameters;
+
+      protected override void Context()
+      {
+         base.Context();
+         _visibleParameter = new PKSimParameter {Visible = true};
+         _hiddenUnchangedParameter = new PKSimParameter {Visible = false, IsDefault = true};
```

Comment:
Our different types of parameters here
Visible
Hidden unchanged
Hidden IsFixedValue = true (Formula was changed to constant....this should result in IsDefault false I believe)
Hidden IsDefault = false